lib/sfrest/group.rb

```diff
@@ -15,6 +15,13 @@ def create_group(groupname)
       @conn.post(current_path, payload)
     end
 
+    # Deletes the group with the specified id
+    # @param [Integer] group_id Id of the group to fetch
+    def delete_group(group_id)
```

Comment:
What is the default behavior if the group is deleted? I know it is defined by the php code but isn't there some questions what  to do with the sites/collections that are in the group?

Reply:
As far as I can tell, the member sites and users are excluded from the group when it is deleted.